ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-general.j2

```diff
@@ -2,6 +2,18 @@
 
 ## Router General
 
```

Comment:
Please delete the blank lines 4, 38 and add a blank line  after 26, 39. 

Also please change your below code to:

```
{%     if router_general.router_id.ipv4 is arista.avd.defined %}

- Global IPv4 Router ID: {{ router_general.router_id.ipv4 }}
{%     endif %}
{%     if router_general.router_id.ipv6 is arista.avd.defined %}

- Global IPv6 Router ID: {{ router_general.router_id.ipv6 }}
{%     endif %}
{%     if router_general.nexthop_fast_failover is arista.avd.defined(true) %}

- Nexthop fast fail-over is enabled.
{%     endif %}
```

This is an old issue but there is a chance of empty lines without the config. So use the above mentioned method of adding blanks above the info.